### DISCUSSION TITLE
how to add space between list numbers and list text?

### DISCUSSION THREAD

**Discussion by nigelwhite** [tailwindlabs/tailwindcss#3690] [Help]
I want the numbers to be spaced away from the text in ordered lists, like this
1.      thing one
2.      thing two

In tailwind.css I have tried adding
`ol li {padding-left: 23px;}`
but this makes no difference.

What is the method to achieve this?
Thanks

**Comment by simonswiss** [comment]
Hey!

`padding-left` on the `li` elements is the way to go: https://play.tailwindcss.com/C3bYLKIlRT

```html
<ol class="list-decimal">
  <li class="pl-4">I am the first item</li>
  <li class="pl-4">I am the second item</li>
  <li class="pl-4">I am the third item</li>
</ol>
```

**Reply by nigelwhite** [reply]
Thanks for this. I need this is tailwind.css as a universal rule because I am styling .md posts created remotely in the Netlify CMS gui. However I've worked out what I was doing wrong. I had ol { list-style-position: inside; } and this was preventing ol li { @apply pl-4 } from working. Cheers

**Reply by simonswiss** [reply]
Nice, glad you've got it sorted 👍
